st3/lsp_utils/npm_client_handler_v2.py

```diff
@@ -180,3 +181,18 @@ def __init__(self, weaksession: 'weakref.ref[Session]') -> None:
 
     def on_ready(self, api: ApiWrapper) -> None:
         pass
+
+    def get_session_buffer_for_uri(self, uri: str) -> Optional[SessionBufferProtocol]:
+        """
+        Return the session buffer for the requested URI.
+
+        :param    uri:  The URI to get session buffer for.
+
+        :returns: An optional session buffer if found for the URI.
+        """
+        session = self.weaksession()
+        if session:
+            return session.get_session_buffer_for_uri_async(uri)
+        return None
```

Comment:
Why not add this to `AbstractPlugin` in LSP?

Reply:
As you see I've dropped this from lsp_utils. No need to create duplication as I can just access that public method from Session directly.